File: templates/leetcode.py
from typing import List


class Solution:
    def problemName(self, nums: List[int], target: int) -> List[int]:
        return [1, 2]


def main() -> None:
    s = Solution()

    # Example Input
    nums: List[int] = [2, 7, 11, 15]
    target: int = 9

    result: List[int] = s.problemName(nums, target)

    # Example Output
    print(f"[{result[0]}, {result[1]}]")


main()

File: templates/leetcode.cpp
#include <iostream>
#include <vector>

using namespace std;

// Solution Code
class Solution {
 public:
  vector<int> twoSum(vector<int>& nums, int target) {}
};
// ----

int main() {
  Solution s;

  // Example Input
  vector<int> nums = {2, 7, 11, 15};
  int target = 9;

  vector<int> result = s.twoSum(nums, target);

  // Example Output
  cout << "[" << result[0] << "," << result[1] << "\n";
  return 0;
}